core/network/fabric-interop-cc/libs/assetexchange/README.md

```diff
@@ -3,4 +3,194 @@
 
  SPDX-License-Identifier: CC-BY-4.0
  -->
-# Asset Exchange Library
\ No newline at end of file
+# Asset Exchange Library
+
+To use this library, in you smart contract `go.mod`, add following in require (update the version according to the latest module version):
+```
+require(
+    ...
+    github.com/hyperledger-labs/weaver-dlt-interoperability/common/protos-go v1.5.3
+    github.com/hyperledger-labs/weaver-dlt-interoperability/core/network/fabric-interop-cc/libs/assetexchange v1.5.3
+    ...
+)
+```
+
+There are two kinds of APIs of library, one with `contractId`, and other without `contractId`.
+
+## With ContractId
```

Comment:
Can we have consistent naming? Is `ContractId` or `contractId` a keyword? If yes, we should use that everywhere with markdown. Otherwise, let's just say "contract ID" in plain English.

Reply:
This was a heading so that's why used ContractId here, everywhere else we have used `contractId`

Reply:
Its not a keyword btw, its just our defined variable name. But also want to avoid confusion of contract ID with smart contract ID